geode-core/src/main/java/org/apache/geode/management/internal/cli/domain/RegionAttributesInfo.java

```diff
@@ -42,11 +42,8 @@
 
 
 public class RegionAttributesInfo implements Serializable {
-  /**
-   *
-   */
-  private static final long serialVersionUID = 1L;
 
+  private static final long serialVersionUID = 336184564012988487L;
```

Comment:
Just curious, why do we need to change to this? if need to be changed, why this complicated number? why not 2L?

Reply:
I believe it's just a convention.  It's not mentioned in any of the Google Java Style Guide that we've adopted as our own.

It needs to be changed to correctly deserialize, but I think you're right -- it could be changed to any not-the-previous-value-`1` and be valid.

Looking around online, there doesn't seem to be any consensus on increment vs rehash the value.  The best I could find was that a big, long number catches the eye and reminds future developers of the importance of considering backwards incompatibility.